napari/layers/shapes/_shapes_models/_polgyon_base.py

```diff
@@ -38,6 +41,8 @@ def __init__(
         filled=True,
         closed=True,
         name='polygon',
+        interpolation_order=1,
```

Comment:
If you want to test locally, you can change this to 3.

Reply:
nice - so 1 means off by default

Reply:
Yup!